collectors/lib/samza_custom_metric_reporter.py

```diff
@@ -17,7 +17,12 @@ def __init__(self, consumer_group_id, kafka_bootstrap_servers, kafka_metrics_top
     def report_samza_custom_metrics(self, metrics_raw, header_raw):
         class_to_metric_name_map = {
             'com.optimizely.sessionization.samza.SessionizationTask' : 'sessionization.metrics',
-            'com.optimizely.preprocessing.samza.enrichevent.Metrics' : 'enrichevents.metrics'
+            'com.optimizely.preprocessing.samza.enrichevent.EnrichProjectIdTask' : 'enrichevents.metrics',
```

Comment:
Adding all the class names seemed easier and efficient, otherwise we would have to iterate over all the keys of `metrics_raw` map.